### PR TITLE
Ignore annotation

### PR DESCRIPTION
**Pull request by albertjan**
supersedes #35

### CODE REVIEW

**Comment by albertjan** [comment]
@simonihmig @Dejavs how does this look?

File: lib/test-runner/mocha/separate/acceptance.js [hunk 1]
```diff
@@ -1,31 +1,41 @@
...
-              let self = this;
-              let promise = new Ember.RSVP.Promise(function(resolve) { yadda.Yadda(library.default(), self).yadda(step, context, resolve); });
-              promise.catch(function() {
-                failed = true;
+    if (feature.annotations.ignore) {
+      this.skip();
+    } else {
+      feature.scenarios.forEach(function(scenario) {
+        describe(`Scenario: ${scenario.title}`, function() {
+          if (feature.annotations.ignore) {
```
Comment (simonihmig) [inline lib/test-runner/mocha/separate/acceptance.js]: Shouldn't this be `if (scenario.annotations.ignore) {` like in the QUnit file?

Comment (albertjan) [inline lib/test-runner/mocha/separate/acceptance.js]: yes it should! good catch.

**Comment by albertjan** [comment]
Alright it works.

**Comment by Dejavs** [comment]
I'm sorry i didn't commented this PR i had hard week, but you're done awesome work! At my company we're using ember-cli-yadda everyday with business ppl and architects. That feature is brilliant!

**Pull request merged**